doc/source/groupby.rst

```diff
@@ -1014,6 +1014,13 @@ Regroup columns of a DataFrame according to their sum, and sum the aggregated on
    df
    df.groupby(df.sum(), axis=1).sum()
 
+Groupby by Indexer to 'resample' data.
```

Comment:
Can you underline this with `~~~~` to make this a header (see a few lines above this for an example)